packages/mui-material/src/ButtonGroup/ButtonGroup.js

```diff
@@ -149,12 +149,18 @@ const ButtonGroupRoot = styled('div', {
         ownerState.color !== 'inherit' && {
           borderColor: theme.palette[ownerState.color].dark,
         }),
+      '&:hover': {
```

Comment:
Don't we need this for the `first-of-type` too?

Reply:
Yes it is for that. Notice that it is within `:not(:last-of-type)`. We don't apply it for last element.

Reply:
Oh man yes! :D